package.json

```diff
@@ -44,7 +44,7 @@
     "@types/mocha": "^2.2.39",
     "@types/mysql": "0.0.33",
     "@types/mz": "^0.0.31",
-    "@types/node": "^6.0.77",
+    "@types/node": "^16.9.1",
```

Comment:
should this be set to the same version of node we are on?

Reply:
Fixed